2016-10-18 17:09:56,107 manager INFO 1.1.0
2016-10-18 17:09:56,108 manager INFO Copyright (C) 2003-2010
2016-10-18 17:09:56,108 manager INFO   Noriaki Ando
2016-10-18 17:09:56,108 manager INFO   Intelligent Systems Research Institute, AIST
2016-10-18 17:09:56,108 manager INFO Manager starting.
2016-10-18 17:09:56,108 manager INFO Starting local logging.
2016-10-18 17:09:56,115 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2016-10-18 17:09:58,171 ManagerServant INFO Master manager not found
2016-10-18 17:09:58,197 manager WARNING Traceback (most recent call last):
  File "C:\Python27\Lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2016-10-18 17:09:58,198 LocalServiceAdmin INFO All the local services are enabled.
2016-10-18 17:09:58,217 ec_base INFO setRate(1000000.000000) done
2016-10-18 17:09:58,219 ec_base INFO setRate(1000.000000) done
2016-10-18 17:09:58,220 rtobject INFO 1 execution context was created.
2016-10-18 17:09:58,230 rtobject INFO Initial active configuration set is default.
2016-10-18 17:10:03,296 SdoServiceAdmin WARNING No available SDO service in the factory: 
2016-10-18 17:10:03,296 SdoServiceAdmin ERROR type  already exists.
2016-10-18 17:10:03,427 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2016-10-18 17:10:03,427 SdoServiceAdmin ERROR type 1f513735-96da-46ec-bf66-ad33ef0ace41 already exists.
2016-10-18 17:17:35,072 ec_worker WARNING ExecutionContext is already stopped.
2016-10-18 17:17:35,072 ec_base ERROR Invoking on_shutdown() for each RTC failed.
